staging/traefik/templates/deployment.yaml

```diff
@@ -11,12 +11,17 @@ metadata:
     chart: {{ template "traefik.chart" . }}
     release: {{ .Release.Name | quote }}
     heritage: {{ .Release.Service | quote }}
-  {{- if .Values.deploymentAnnotations }}
+    {{- if .Values.deployment.labels }}
+    {{- toYaml .Values.deployment.labels | nindent 4 }}
+    {{- end }}
+  {{- with .Values.deployment.annotations }}
```

Comment:
We should document this change in the release-notes, users might be using this option.

Reply:
Added in release notes section